test/svg.jl

```diff
@@ -54,6 +54,7 @@ end
     Object((args...) -> latex(L"8", O + Point(20, 20), :bottom, :right))
     Object((args...) -> latex(L"8", O + Point(20, 20), :bottom, :left))
     Object((args...) -> latex(L"8", O + Point(20, 20), :top, :right))
+    Object((args...) -> latex(L"8", O + Point(20, 20), valign = :top, halign = :right))
```

Comment:
The line before should be removed, right?

Reply:
I am not sure why it should be removed. i was thinking the explicit arguments were a different test.

Reply:
Now it's drawing the same text twice at the same position.

Reply:
Ok, removed it.